packages/create-react-admin/src/generateAppTestFile.ts

```diff
@@ -20,6 +20,9 @@ test("should pass", async () => {
             ? `
 
 	// Sign in
+  	// Ensure the form is fully loaded before interacting with it
+  	await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
You should use an assertion in a waitfor instead of a timeout. Same for the other 2 instances

Reply:
We already do. I still had to add this

Reply:
Instead, add a timeout option to the initial `findByLabelText`, as it proxies its options to `waitFor`.